src/main/java/com/segment/analytics/android/integrations/localytics/LocalyticsIntegration.java

```diff
@@ -134,16 +134,35 @@
       Localytics.setCustomerId(userId);
       logger.verbose("Localytics.setCustomerId(%s);", userId);
     }
+
     String email = traits.email();
     if (!isNullOrEmpty(email)) {
       Localytics.setIdentifier("email", email);
+      Localytics.setCustomerEmail(email);
       logger.verbose("Localytics.setIdentifier(\"email\", %s);", email);
+      logger.verbose("Localytics.setCustomerEmail(\"$email\", %s);", email);
```

Comment:
Should be just `logger.verbose("Localytics.setCustomerEmail(%s);", email);`. We're just logging `Localytics.setCustomerEmail(email);`